inst/schema/spec.md

```diff
@@ -0,0 +1,229 @@
+This is all highly uncertain and liable to change!
+
+# Data
+## POST /data/tab-name/
+Accepts a set of baseline options, and returns a data set for the data under the named tab.
+
+Request schema: [DataOptions.schema.json](./DataOptions.schema.json)
+
+Return schema: [Data.schema.json](./Data.schema.json)
+
+### Example
+#### Request
+```
+{
+  "irs_future": "80%",
+  "sprayInput": 1,
+  "biting_indoors": "high",
+  "population": 1000
+}
+```
+
+#### Response
+```
+[
+    {
+        "intervention": "none",
+        "net_use": 20,
+        "prev_year_1": 7.32,
+        "prev_year_2": 7.34,
+        "prev_year_3": 7.48,
+        "cases_averted": 0,
+        "cases_averted_high": 0,
+        "cases_averted_low": 0,
+    },
+    {
+        "intervention": "none",
+        "net_use": 40,
+        "prev_year_1": 7.32,
+        "prev_year_2": 7.34,
+        "prev_year_3": 7.48,
+        "cases_averted": 0,
+        "cases_averted_high": 0,
+        "cases_averted_low": 0,
+    },
+    {
+        "intervention": "ITN",
+        "net_use": 20,
+        "prev_year_1": 5.32,
+        "prev_year_2": 5.34,
+        "prev_year_3": 5.48,
+        "cases_averted": 51,
+        "cases_averted_high": 60,
+        "cases_averted_low": 50,
+    },
+    {
+        "intervention": "ITN",
+        "net_use": 40,
+        "prev_year_1": 4.32,
+        "prev_year_2": 4.34,
+        "prev_year_3": 4.48,
+        "cases_averted": 52,
+        "cases_averted_high": 62,
+        "cases_averted_low": 50,
+    },
+    {
+        "intervention": "PBO",
+        "net_use": 20,
+        "prev_year_1": 6.32,
+        "prev_year_2": 6.34,
+        "prev_year_3": 6.48,
+        "cases_averted": 61,
+        "cases_averted_high": 70,
+        "cases_averted_low": 60,
+    },
+    {
+        "intervention": "PBO",
+        "net_use": 40,
+        "prev_year_1": 5.32,
+        "prev_year_2": 5.34,
+        "prev_year_3": 5.48,
+        "cases_averted": 62,
+        "cases_averted_high": 72,
+        "cases_averted_low": 60,
+    }
+]
+```
+
+# Tables
+## GET /table/:tab-name/
```

Comment:
Again not sure if there's any point in separating these metadata endpoint by tab, or whether it should just be one endpoint. It's not like the responses are very big, so maybe makes sense to just have /tables/ and /graphs/, with some indication of which tab the individual items belong to?